docs/scripts/wait-for-flyway.sh

```
@@ -0,0 +1,65 @@
+      echo "Waiting... Flyway migration ${REQUIRED_FLYWAY_VERSION} not found or not successful yet. Status: [${migration_status}]"
+    fi
+
+    sleep ${WAIT_INTERVAL}
+  done
+
+  # Unset PGPASSWORD for security after use (optional)
+  unset PGPASSWORD
+fi # End of conditional wait block
+
+# --- Execute the original command ---
+echo "Proceeding to execute command: $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)